packages/contracts-bedrock/contracts/test/Hashing.t.sol

```diff
@@ -22,15 +23,17 @@ contract Hashing_Test is CommonTest {
 
     function test_hashDepositTransaction() external {
         bytes32 digest = Hashing.hashDepositTransaction(
-            0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266,
-            0xB79f76EF2c5F0286176833E7B2eEe103b1CC3244,
-            0xde0b6b3a7640000,
-            0xe043da617250000,
-            0x2dc6c0,
-            false,
-            hex"",
-            0xd25df7858efc1778118fb133ac561b138845361626dfb976699c5287ed0f4959,
-            0x1
+            Types.UserDepositTransaction(
```

Comment:
We need to delete this test, we have coverage via differential testing + the ci devnet deposit test covers this

Reply:
Note that the hashing scheme changed so this test should fail if you rebase